addon/components/base/bs-accordion/item.js

```diff
@@ -18,6 +18,11 @@ import layout from 'ember-bootstrap/templates/components/bs-accordion/item';
 export default Component.extend(TypeClass, {
   layout,
 
+  /**
+   * We override the mixin here to include an extra class.
+   * @see TypeClass
+   */
+  classNameBindings: ['typeClass', 'disabled'],
```

Comment:
I don't think you need to add `typeClass` here, as `classNameBindings` will merge the items used in parent classes/mixins, IIRC

Reply:
I didn't realize mixins magically combined merged those together...that's some interesting "magic"

Reply:
Yeah, Ember has some places of extraordinary magic indeed! ;) 

I guess some of these are maybe going to go away some day, when moving to native classes etc. Here for the record: https://emberjs.com/api/ember/3.4/classes/EmberObject/properties/mergedProperties?anchor=mergedProperties